### DISCUSSION TITLE
Firebase push notification question - begginer

### DISCUSSION THREAD

**Discussion by Yahya0007** [shinyorg/shiny#1376] [Q&A] [answered]
Hi

MAUI net 8.0.

I have setup code as per docs; https://shinylib.net/client/push/

I have added the Firebase google-services.json for Android to both MAUI project root folder and the MAUI Project\Platforms\Android folders.

Running the MAUI project does not call either the CheckPermission() or the OnTokenRefreshed methods.

What are some of things I can check to see where the block is? Thanks.

Thanks

Regards

**Comment by aritchie** [comment, marked as answer]
I have added additional documentation for push.

CheckPermission() is a sample method.  You need to make the calls yourself.

**Comment by Yahya0007** [comment]
Hi Allan

This worked nicely and I am receiving notifications in Android. Many Thanks. 

Regards
